android/jni/com/mapswithme/core/jni_helper.hpp

```diff
@@ -50,5 +50,20 @@ namespace jni
   jobject GetNewPoint(JNIEnv * env, m2::PointD const & point);
   jobject GetNewPoint(JNIEnv * env, m2::PointI const & point);
 
+  template<typename TValue, typename TToJavaFn>
+  jobjectArray ToJavaArray(JNIEnv * env, jclass clazz, vector<TValue> const & src, TToJavaFn && toJavaFn)
```

Comment:
+, заменил везде по проекту создания массивов.
> On Apr 28, 2016, at 12:00 PM, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/jni/com/mapswithme/core/jni_helper.hpp <https://github.com/mapsme/omim/pull/3042#discussion_r61394585>:
> 
> > @@ -50,5 +50,20 @@ namespace jni
> >    jobject GetNewPoint(JNIEnv * env, m2::PointD const & point);
> >    jobject GetNewPoint(JNIEnv * env, m2::PointI const & point);
> >  
> > +  template<typename TValue, typename TToJavaFn>
> > +  jobjectArray ToJavaArray(JNIEnv * env, jclass clazz, vector<TValue> const & src, TToJavaFn && toJavaFn)
> В Editor.cpp есть функция с тем же именем. Выпилил её?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3042/files/4ec789b43f920686cf11edafc48c5e58449ba38a#r61394585>